flask-backend/api/routes/search.py

```diff
@@ -0,0 +1,123 @@
+import sys
```

Comment:
Add comment at the top of the file describing the purpose, so that general contributors can directly understand just by looking at it.
Also add comments to every function.
remove redundant print statements.

Reply:
@shivanshu1333 sir! added comments. 🙂

Reply:
Format the code properly as in follow python conventions to write the code.
There's a lot of redundant space in between lines, use a linter if needed.

Reply:
Deleted some lines. Now I think deleting more lines will not look better in reading.

Reply:
install a python linter in your system, and it will format the code for you. It might remove some of the comments, replace those comments back to your code.